src/main/java/frc/robot/RobotContainer.java

```diff
@@ -42,6 +43,9 @@ public RobotContainer() {
    * {@link edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
   private void configureButtonBindings() {
+    drivetrain.setDefaultCommand(new TeleopTankDrive(drivetrain, 
+    () -> -xbox.getY(Hand.kLeft),
+    () -> xbox.getX(Hand.kRight)));
```

Comment:
[Continuation lines should be indented by four spaces.](https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent)